src/components/WatchlistHistory/constants.js

```diff
@@ -0,0 +1,21 @@
+import { DAY, getTimeIntervalFromToday, MONTH } from '../../utils/dates'
+
+export const TOTAL_TYPES = {
+  all: 'TOTAL_MARKET',
+  erc20: 'TOTAL_ERC20'
+}
+
+const RANGES_METHOD = new Map()
```

Comment:
Do we have this in our source in other place?

Reply:
@Partysun @haritonasty [`getIntervalByTimeRange`](https://github.com/santiment/app/blob/3a6d4a9efbc088fdd5086dd76c6a77e15cb010c1/src/utils/dates.js#L280) is our parser for such cases. Returns `{from: Date, to: Date}`